identity-wallet/src/state/connections/reducers/handle_siopv2_authorization_request.rs

```diff
@@ -64,10 +64,7 @@ pub async fn handle_siopv2_authorization_request(mut state: AppState, _action: A
             connection.last_interacted = connection_time.clone();
         });
 
-    // TODO: This is a HORRIBLE solution to determine the connection_id by the non-unique "issuer name".
-    // It is a TEMPORARY solution and should only be used in DEMO environments,
-    // since we currently lack a unique identitfier to distinguish connections.
-    let connection_id = base64::encode_config(&client_name, base64::URL_SAFE);
```

Comment:
`handle_oid4vp_authorization_request.rs` has the same TODO message so It would be nice to replace that there as well :)